File: algorithms/1913.maximum-product-difference-between-two-pairs/solution.go
func maxProductDifference(nums []int) int {
	l := len(nums)
	sort.Slice(nums, func(i, j int) bool {
		return nums[i] < nums[j]
	})

	return nums[l-1]*nums[l-2] - nums[0]*nums[1]
}

File: algorithms/1913.maximum-product-difference-between-two-pairs/solution.rs
impl Solution {
    pub fn max_product_difference(nums: Vec<i32>) -> i32 {
        let len = nums.len();
        let mut nums = nums;
        nums.sort_unstable();

        nums[len - 1] * nums[len - 2] - nums[0] * nums[1]
    }
}
